packages/headless/src/controllers/tab/headless-tab.ts

```diff
@@ -37,8 +43,9 @@ export interface TabInitialState {
   isActive: boolean;
 }
 
-const optionsSchema = new Schema({
-  expression: new StringValue(),
+const optionsSchema = new Schema<Required<TabOptions>>({
+  expression: new StringValue({required: true, emptyAllowed: true}),
+  id: new StringValue({required: false, emptyAllowed: false}),
```

Comment:
We could set the default to `default` ... 🤔  It matches what we set  for origin level 2 by default.

And we could change the warning message to verify that, if you use a tab, you should set it to something else than 'default' ?

Reply:
Sounds like an improvement 👍

Reply:
That would be an improvement 👍